server/mdm/apple/vpp/api.go

```diff
@@ -184,6 +191,73 @@ func GetAssets(token string, filter *AssetFilter) ([]Asset, error) {
 	return bodyResp.Assets, nil
 }
 
+type AssignmentFilter struct {
```

Comment:
Can we also add the link to the Apple docs as a comment here (it'll be the same as the one for `GetAssignments`, I think)?

Reply:
Good call!